### ISSUE TITLE
Wide screen terminal

### ISSUE THREAD

**Issue by nolageek** [opicron/mysticbbs-rjam#3]
Would love support for 132x37 terminal connections.

**Comment by opicron** [comment]
Absolutely, I need to use the new Mystic a48 functions which can find the correct ini files based on the current terminal size. Its definately on the todo list!

**Comment by nolageek** [comment]
Awesome!

**Comment by maliciouspastry** [comment]
Yes please, this script alone makes reading messages bearable. Wide screen support would be amazing!

**Comment by opicron** [comment]
Ok guys I made a start with loading INI files the new Mystic way. Now I have to replace each individual setting in the code. Will continue to do so in the coming weeks :). But wide-screen suppport will be done afterwards. As everything is already configurable, do not need to change core code for that.

**Comment by opicron** [comment]
I have implemented widescreen a bit on my board, TheForze. Could you please check rJam there and see if this is working for you and if you agree with the direction its taking?

**Comment by maliciouspastry** [comment]
> I have implemented widescreen a bit on my board 132x37, TheForze. Could you please check rJam there and see if this is working for you and if you agree with the direction its taking?

I tried connecting to your board, but it asks for a username, does not seem to allow a new account. Maybe I am doing something wrong XD. I'm running mine at 132x52 and most things seem to work great, if this is working then that would be amazing.

Either way, as it is (my version) with 132xANY it just really messes up. If you got it listing messages, browsing, and reading its probably what we are really looking for. The editor uses the Mystic one which works fine (at least for me, I am sure for others too.)

**Comment by opicron** [comment]
Yes the bord should function fine, it asks for a username. If the username exists it requests a password, else a new user notice pops up.

2 days ago there was a small issue with login. I fixed that :). Would you mind checking it out now?

**Comment by maliciouspastry** [comment]
I finally got in, apparently I registered at some point but no idea, made a new account.

So yeah, that's working perfectly (I tested in 132x52) from what I tested on the system and you got the left right doing the same as the files version of the mod which is amazing. It's not cutting off and losing track of the lines anymore.

Is this a version your able to push to the Git here?

**Comment by opicron** [comment]
I will push the revision onto git shortly.

**Comment by maliciouspastry** [comment]
Nice. Always great to have some new things to play with.

**Comment by maliciouspastry** [comment]
Any updates bud?

**Comment by opicron** [comment]
Yes I polished the mod, should be ready. Need ansi art for release.

**Comment by opicron** [comment]
I updated the github with the latest version! Looking forward to your findings ^^.
